infinite_tracing/test/test_helper.rb

```diff
@@ -57,6 +57,8 @@ def with_serial_lock
   CLIENT_MUTEX.synchronize { yield }
 end
 
+# This will always be false unless you physically flip it to true
+# Setting this to true gives extra trace information
```

Comment:
This change is unrelated to gRPC and was discovered while investigating the way we tested gRPC in infinite tracing, which uses the library.